### PR TITLE
Move Sophia to core

### PR DESCRIPTION
**Pull request by soapy1**
PR Checklist:

- [x] note any issues closed by this PR with [closing keywords](https://help.github.com/articles/closing-issues-using-keywords)
- [x] if you are adding a new page under `docs/` or `community/`, you have added it to the sidebar in the corresponding `_sidebar.json` file
- [x] put any other relevant information below

### CODE REVIEW

File: src/core.csv [hunk 1]
```diff
@@ -20,6 +20,7 @@ pkgw,peter@newton.cx,Peter K. G. Williams
 SylvainCorlay,sylvain.corlay@gmail.com,Sylvain Corlay
 synapticarbors,joshua.adelman@gmail.com,Joshua Adelman
 sodre,psodre@gmail.com,Patrick Sodré
+soapy1,scastellarin@anaconda.com,Sophia Castellarin
```
Comment (beckermr) [inline src/core.csv:23]: Can you update this email @soapy1?

Comment (soapy1) [reply src/core.csv]: good catch!

**Review by jaimergp — APPROVED** [review]

**Comment by beckermr** [comment]
@soapy1 should merge to ensure she still has the proper permissions!

**Comment by jakirkham** [comment]
Welcome back! 😄

**Pull request merged**